babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -788,15 +788,70 @@ SqlIndex SqlCreateTableIndex(Span s) :
 SqlNode SqlExecMacro() :
 {
     SqlIdentifier macro;
+    SqlNodeList paramNames = new SqlNodeList(getPos());
+    SqlNodeList paramValues = new SqlNodeList(getPos());
     Span s;
 }
 {
     macro = CompoundIdentifier() { s = span(); }
+    [SqlExecMacroArgument(paramNames, paramValues)]
     {
-        return new SqlExecMacro(s.end(this), macro);
+        return new SqlExecMacro(s.end(this), macro, paramNames, paramValues);
     }
 }
 
+void SqlExecMacroArgument(SqlNodeList paramNames, SqlNodeList paramValues) :
+{
+    SqlNode name;
+    SqlNode value;
+}
+{
+    LOOKAHEAD(2)
+    <LPAREN>
+    (
+        value = Literal()
+        {
+            paramValues.add(value);
+        }
+
+    )
+    (
+        <COMMA>
+        value = Expression(ExprContext.ACCEPT_SUB_QUERY)
+        {
+            paramValues.add(value);
+        }
+
+    )*
+    <RPAREN>
+    |
```

Comment:
It could help readability to wrap everything for this "or" statement in parentheses and indent everything besides the "|" over one tab

Reply:
Good point. Fixed in https://github.com/googleinterns/calcite/pull/108/commits/6bc3ba6bc336c390bc38e8c7508b7af19dbc7ee2